### ISSUE TITLE
reignore erroneously duplicates the last part of a file

### ISSUE THREAD

**Issue by NickHeiner** [airbnb/ts-migrate#94]
I'm seeing one of my files get erroneously modified by the codemod.

```
$ ts-migrate reignore .
TypeScript version: 4.1.2
Initialized tsserver project in 14353.990ms.
Start...
[strip-ts-ignore] Plugin 1 of 3. Start...
[strip-ts-ignore] Finished in 2214.610ms.
[ts-ignore] Plugin 2 of 3. Start...
[ts-ignore] Finished in 507805.555ms.
[eslint-fix-changed] Plugin 3 of 3. Start...
[eslint-fix-changed] Finished in 14272.833ms.
Finished in 524295.523ms, for 3 plugin(s).
Writing 420 updated file(s)...
Wrote 420 updated file(s) in 87.555ms.
```

Problematic file, before it was modified:
```ts
/** 100 lines omitted from this GH issue */
export function gridList(itemArea: Area, itemsPerRowOrColumn: number) {
    return new GridBuilder(itemArea, itemsPerRowOrColumn);
}

export function variableItemList(minItemArea: Area) {
    return new MinItemBuilder(minItemArea);
}

```

After transformation:
```ts
/** 100 lines omitted from this GH issue */
export function gridList(itemArea: Area, itemsPerRowOrColumn: number) {
    return new GridBuilder(itemArea, itemsPerRowOrColumn);
}

export function variableItemList(minItemArea: Area) {
    return new MinItemBuilder(minItemArea);
}
er) {
    return new GridBuilder(itemArea, itemsPerRowOrColumn);
}

export function variableItemList(minItemArea: Area) {
    return new MinItemBuilder(minItemArea);
}

```

Starting from the `er) {` line, you can see the end of the file has been repeated.

I tried to create a minimal repro of this outside of my repo but was unable to.

I tried resetting the files and rerunning the command, and got the same result.

**Comment by NickHeiner** [comment]
As a workaround, I can just manually fix the impacted file. As long as this error creates syntax errors, `tsc` easily identifies them.

**Comment by Rudeg** [comment]
Hi @NickHeiner, I tried to run reignore on the code sample above, but it worked fine for me :(

**Comment by NickHeiner** [comment]
Yeah, I tried that and also I tried running it on a larger code sample, and I couldn't repro. But it does consistently repro on my full repo.

I may spend a bit of time trying to figure out what's going wrong. But, I can also easily work around this by fixing it manually, so I probably won't spend too long on it. But by posting here, we'll at least have a reference if someone else encounters the same issue. :)
